package.json

```
@@ -118,11 +124,23 @@
     "start:proxy": "PROXY=true npm run start",
     "start:proxy:beta": "BETA=true npm run start:proxy",
     "travis:build": "NODE_ENV=production webpack --config config/test.webpack.config.js",
-    "travis:verify": "npm-run-all build:prod lint test",
-    "verify": "npm-run-all build lint test",
-    "nightly": "npm run travis:verify"
+    "travis:verify": "npm-run-all build:prod lint test:ct",
+    "verify": "npm-run-all build lint test:ct",
```

```suggestion
    "verify": "npm-run-all build lint test:jest",
```
This would fix your error in pr_check. we don't want to run cypress tests there.